pep-3156.txt

```diff
@@ -361,7 +373,8 @@ stopping and closing.  (However, a partially-conforming event loop is
 still better than nothing. :-)
 
 - Starting, stopping and closing: ``run_forever()``,
-  ``run_until_complete()``, ``stop()``, ``is_running()``, ``close()``.
+  ``run_until_complete()``, ``stop()``, ``is_running()``, ``close()``,
```

Comment:
I'm not sure I understand the "racy" comment. Are you saying they are attractive to people who mix threads and event loops? Pretty much every API is racy in that context. When you're just talking about a single thread I see nothing racy about these APIs -- is_running() basically distinguishes between "you are in a coroutine and your loop is fine" and "you are in synchronous code and you should should be managing the loop (using run_until_complete() or run_forever())", whereas is_closed() can only be true in the latter mode and then tells you that your loop is dead. Right?